convert-sg-db.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+
+import json
+import commonmark
+
+
+def create_biblio(db):
+    biblio = {}
+
+    for workstream in db["workstreams"]:
+        for standard in workstream["standards"]:
+            reference, contents = standard_or_idea_to_biblio(standard)
+            contents["title"] += " Standard"
+            biblio[reference] = contents
+
+    for idea in db["ideas"]:
+        reference, contents = standard_or_idea_to_biblio(idea)
+        contents["status"] = "Living Idea"
+        biblio[reference] = contents
+
+    for biblio_item in db["biblio"]:
+        reference, contents = biblio_item_to_biblio(biblio_item)
+        biblio[reference] = contents
+
+    string = json.dumps(biblio, ensure_ascii=False, allow_nan=False, sort_keys=True, indent=2)
+    string += "\n"
+    open("resources.whatwg.org/biblio.json", "w").write(string)
+
+def standard_or_idea_to_biblio(document):
+    return (document["reference"], {
+        "authors": [author_to_biblio(author) for author in document["authors"]],
+        "href": document["href"],
+        "title": document["name"]
+    })
+
+def author_to_biblio(person):
+    return person["name"]
```

Comment:
Nit: move after `biblio_item_to_biblio` to keep `biblio_item_to_biblio` and `standard_or_idea_to_biblio` together?